gemma-core/src/main/java/ubic/gemma/persistence/util/SpringProfiles.java

```diff
@@ -0,0 +1,13 @@
+package ubic.gemma.persistence.util;
+
+/**
+ * Constants used for various Spring profiles.
+ * @author poirigui
+ */
+@SuppressWarnings("unused")
+public class SpringProfiles {
+
+    public static final String PRODUCTION = "production",
+            DEV = "dev",
+            TEST = "test";
+}
```

Comment:
These can be used within the code as constants for toggling profile-specific features.